packages/components/src/internal/components/ontology/OntologyTreeSearchContainer.tsx

```diff
@@ -0,0 +1,155 @@
+                    });
+            }, 500);
+
+            return () => clearTimeout(timeOutId);
+        }
+    }, [searchTerm, setError, setSearchHits, setTotalHits]);
+
+    const onItemClick = useCallback(
+        async (evt: MouseEvent<HTMLLIElement>, code: string) => {
+            // for now we will just send the user to the first path for this concept, in the future we'll add in UI
+            // that lets the user select if more then one path exists for the concept
+            const codePaths = await fetchAlternatePaths(code);
+            if (codePaths?.length > 0) {
+                searchPathClickHandler(codePaths[0], true);
+            }
+        },
+        [searchPathClickHandler]
+    );
+
+    return (
+        <div className="concept-search-container">
+            <form autoComplete="off">
+                <input
+                    type="text"
+                    className="form-control"
+                    name="concept-search"
+                    placeholder={'Search ' + ontology.abbreviation}
+                    onChange={onSearchChange}
+                    onFocus={onSearchFocus}
+                    onBlur={onSearchBlur}
+                />
+            </form>
+            <OntologySearchResultsMenu
+                searchHits={searchHits}
+                totalHits={totalHits}
+                isFocused={isFocused}
+                error={error}
+                onItemClick={onItemClick}
+            />
+        </div>
+    );
+});
+
+interface OntologySearchResultsMenuProps {
+    searchHits: ConceptModel[];
+    totalHits: number;
+    isFocused: boolean;
+    error: string;
+    onItemClick: (evt: MouseEvent<HTMLLIElement>, code: string) => void;
+}
+
+// exported for jest testing
+export const OntologySearchResultsMenu: FC<OntologySearchResultsMenuProps> = memo(props => {
+    const { searchHits, isFocused, totalHits, error, onItemClick } = props;
+    const showMenu = useMemo(() => isFocused && (searchHits !== undefined || error !== undefined), [isFocused, searchHits, error]);
+    const hitsHaveDescriptions = useMemo(() => searchHits?.findIndex(hit => hit.description !== undefined) > -1, [searchHits]);
+
+    if (!showMenu) {
+        return null;
+    }
+
+    return (
+        <div>
+            <ul className="result-menu container">
+                <Alert>{error}</Alert>
+                {searchHits?.length === 0 && (
+                    <li key="none">
+                        <div className="row">
+                            <div className="col col-xs-12">No search results found.</div>
+                        </div>
+                    </li>
+                )}
+                {searchHits?.map(hit => (
+                    <li
+                        key={hit.code}
+                        className="selectable-item"
+                        role="option"
+                        onMouseDown={evt => onItemClick(evt, hit.code)}
```

Comment:
should probably be a callback function since this is in a loop

Reply:
I needed to get the specific `hit.code` for the given `searchHit`, so I left this inline for now. We can refactor later if needed.

Reply:
i think this would work:

```
    const hitClickHandler = (evt: MouseEvent<HTMLLIElement>): void => {
        onItemClick?.(evt, evt.target['key']);
    };
```

also is there a reason to use the onMouseDown event vs onClick?

Reply:
I believe I tried that but I wasn't able to access that `key` prop off of the `evt.target`. I can try again, but in the interest of time for the 21.3 release, I will merge this as is for now.

Reply:
Regarding your second question, I thought `onClick` would work as well, but it wasn't getting triggered. I believe it has something to do with the fact that this is an `li` element.